docs/guides/expectations/contributing/how-to-contribute-a-new-expectation-to-great-expectations.md

```diff
@@ -142,79 +144,103 @@ examples = [{
         }
     ],
 }]
-The value of examples is a list of examples.
+````
+
+The value of `examples` is a list of examples.
 
 Each example is a dictionary with two keys:
 
-data: defines the input data of the example as a table/data frame. In this example the table has one column named “mostly_threes” with 10 rows. If you define multiple columns, make sure that they have the same number of rows. If possible, include test data and tests that includes null values (None in the python test definition).
-tests: a list of test cases that use the data defined above as input to validate
+* `data`: defines the input data of the example as a table/data frame. In this example the table has one column named “mostly_threes” with 10 rows. If you define multiple columns, make sure that they have the same number of rows. If possible, include test data and tests that includes null values (None in the python test definition).
+
+* `tests`: a list of test cases that use the data defined above as input to validate
 
-title should be a descriptive name for the test case. Make sure to have no spaces.
+	* `title` should be a descriptive name for the test case. Make sure to have no spaces.
```

Comment:
<img width="974" alt="Screen Shot 2021-07-09 at 4 59 43 PM" src="https://user-images.githubusercontent.com/49923762/125135603-4549ff80-e0d7-11eb-956f-0bdc7b9574ec.png">
c4a2313b7e64.png">
<img width="725" alt="Screen Shot 2021-07-09 at 5 03 23 PM" src="https://user-images.githubusercontent.com/49923762/125135779-92c66c80-e0d7-11eb-87dc-6a467ae486be.png">

I believe these should be rendered as  as opposed to code blocks. Seems like some finicky markdown issue.